bears/c_languages/GNUIndentBear.py

```diff
@@ -23,18 +23,169 @@ class GNUIndentBear:
     def create_arguments(filename, file, config_file,
                          max_line_length: int=80,
                          use_spaces: bool=True,
+                         blank_lines_after_declarations: bool=False,
```

Comment:
maybe it should, i guess its okay for now, maybe we should open an issue to order all params in lint bears alphabetically.